<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplingProfilePlotter</class>
 <widget class="QWidget" name="SamplingProfilePlotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sampled profile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Graph</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QComboBox" name="y">
         <property name="currentIndex">
          <number>7</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Stimulation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛒 weighted</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛒 unweighted</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛒 target only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛁 weighted</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛁 unweighted</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛁 target only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fitness</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Score</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="x">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Stimulation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛒 weighted</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛒 unweighted</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛒 target only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛁 weighted</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛁 unweighted</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>𝛁 target only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fitness</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Score</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="hideUnselected">
         <property name="text">
          <string>Hide unselected</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>x:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>y:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QPushButton" name="showAll">
         <property name="text">
          <string>Show all</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QPushButton" name="pdf">
         <property name="text">
          <string>PDF...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="9">
        <widget class="QCustomPlot" name="plot" native="true"/>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="pareto">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selects the pareto front along the dimensions checked in the table.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Select pareto front</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Table</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="2">
        <widget class="QPushButton" name="addDeckGo">
         <property name="text">
          <string>Go</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="addDeckCombo">
         <item>
          <property name="text">
           <string>New deck</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Add selected stimulation to deck...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="table">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>90</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Stimulation</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>𝛒 weighted</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>𝛒 unweighted</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>𝛒 target only</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>𝛁 weighted</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>𝛁 unweighted</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>𝛁 target only</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fitness</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Score</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Pareto front and score dimensions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>𝛒 normalisation interval for scoring:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="rho_normalisation">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>[-1, max(𝛒)]</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>[min(𝛒), max(𝛒)]</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>[min(𝛒 ∪ {0}), max(𝛒)]</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>[floor(𝛒), max(𝛒)]</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableWidget" name="paretoDims">
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <row>
          <property name="text">
           <string>𝛒 weighted</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>𝛒 unweighted</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>𝛒 target only</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>𝛁 weighted</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>𝛁 unweighted</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>𝛁 target only</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Fitness</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Minimise</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Maximise</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Not included</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Include in score</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="profile"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
